.github/workflows/go.yml

```diff
@@ -5,7 +5,6 @@ name: Go
 
 on:
   push:
-    branches: ["main"]
```

Comment:
Please keep this one, I want to make sure main is OK at any moment!

Reply:
Same behavior as before.  

see: https://github.com/commander-cli/cmd/blob/main/.github/workflows/lint.yaml#L3 for a reference. We can keep it explicit if you want.

Reply:
This isn't the exact same setup than what's proposed here

<img width="287" alt="image" src="https://github.com/go-fuego/fuego/assets/46993939/ae2e7e78-56a5-4263-94b3-39f3cc4494ab">